src/app/widgets/button/button.widget.js

```diff
@@ -125,7 +125,7 @@
                     break;
 
                 case "toggle":
-                    if (vm.widget.command && vm.widget.command_alt) {
+                    if (vm.widget.command != null && vm.widget.command_alt != null) {

```

Comment:
or maybe we remove this check completely? why is it needed?

Reply:
with thsi check now, it is not possible fro example to toggle between a string and null

Reply:
(okay this was also not possible before)